package.json

```diff
@@ -1,6 +1,8 @@
 {
   "activationEvents": [
     "onLanguage:yaml",
+    "onCommand:extension.ansible.navigate",
```

Comment:
My impression was that we would use "navigator" everywhere and not "navigate". 

@ganeshrn WDYT?

Reply:
Ye agreed the name should be `navigator` since the config file is for `ansible-navigator` tool

Reply:
But since it's a command/action/function/callback, it should be a verb. The command is `ansible-navigator run` after all, with "run".

Reply:
`ansible.ansibleLint.path` is used for ansible-lint executable path and `ansible.ansible.path` for ansible excutable. 
I think the naming convention should follow a uniform pattern. How about `ansible.ansibleNavigator.path` and `ansible.ansiblePlaybook.path`?

Reply:
I've changed these already. But it seems like you're confusing the commands (verbs, they are essentially actions that users invoke) and the settings (just values, like an executable path).

This specific place defines the commands.

Reply:
I added the comment at wrong line, reading at correct line number